.claude/commands/fix-cve.md

```
@@ -0,0 +1,276 @@
+
+**Execute ALL 10 steps automatically:**
+
+### Step 1: Find Current Sprint
+```
+Tool: searchJiraIssuesUsingJql
+jql: project = RHAIENG AND component = Notebooks AND sprint in openSprints()
+```
+→ `✓ Sprint: Notebooks - Narwal`
+
+### Step 2: Get Developer Account ID
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)